pkg/distributor/push_test.go

```diff
@@ -1178,7 +1186,7 @@ func TestHandler_ToHTTPStatus(t *testing.T) {
 			)
 			require.NoError(t, err)
 
-			status := toHTTPStatus(ctx, tc.err, limits)
+			_, status := toGRPCHTTPStatus(ctx, tc.err, limits)
```

Comment:
We should be asserting the gRPC code too if we're using the same method to generate them and HTTP codes. Either that or we should split the error translation methods.